File: 217-contains-duplicate/contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let hashMap = new Set();

    for (let i=0; i<nums.length; i++) {
        if (hashMap.has(nums[i])) {
            return true;
        } 
        hashMap.add(nums[i])
    }

    return false
    
};

File: 217-contains-duplicate/contains-duplicate.java
import java.util.HashSet;
import java.util.Set;
class Solution {
    public boolean containsDuplicate(int[] nums) {

        Set <Integer> uniqueSet = new HashSet <> ();

        for (int i=0; i<nums.length; i++) {
            if (uniqueSet.contains(nums[i])) {
                return true;
            } else {
                uniqueSet.add(nums[i]);
            }
        }

        return false;
        
    }
}